store.html

```diff
@@ -0,0 +1,163 @@
+                    </div>
+                    <div>
+                        <button class="btn btn-bg">Place my order</button>
+                    </div>
+                </form>
+            </div>
+            <div class="order-section-2"></div>
+        </section>
+        -->
+
+        <section class="order-section">
+            <div class="order-section-1">
+                <h2>Order your Karma wifi device today!</h2>
+                <form>
+                    <div class="form-c-first-name">
+                        <label for="first-name">First name *</label>
+                        <input id="first-name" type="text" name="first-name" minlength="3" maxlength="10" required />
+                    </div>
+                    <div class="form-c-last-name">
+                        <label for="last-name">Last name *</label>
+                        <input id="last-name" type="text" name="last-name" minlength="3" maxlength="10" required />
+                    </div>
+                    <div class="form-c-address-1">
+                        <label for="address">Address *</label>
+                        <input id="address" type="text" name="address" minlength="3" maxlength="50" required />
+                    </div>
+                    <div class="form-c-address-2">
+                        <label for="address-2">Address 2</label>
+                        <input id="address-2" type="text" name="address-2" minlength="3" maxlength="50" />
+                    </div>
+                    <div class="form-c-city">
+                        <label for="city">City</label>
+                        <select id="city" type="select" name="city" required>
+                            <option value="">Select your city...</option>
+                            <option value="manchester">Manchester</option>
+                            <option value="leeds">Leeds</option>
+                        </select>
+                    </div>
+                    <div class="form-c-post-code">
+                        <label for="post-code">Post Code *</label>
+                        <input id="post-code" type="text" name="post-code" minlength="3" maxlength="10" required />
+                    </div>
+                    <div class="form-c-color">
+                        <fieldset>
+                            <legend>Select a colour</legend>
+                            <input id="orange" type="radio" name="color" value="orange" required />
+                            <label for="orange">Karma orange</label>
+                            <input id="grey" type="radio" name="color" value="grey" required />
+                            <label for="grey">Space Grey</label>
+                        </fieldset>
+                    </div>
+                    <div class="form-c-terms">
+                        <input class="form-inline" id="terms" type="checkbox" name="terms" value="agree" required />
```

Comment:
The check box should have (checked) attribute instead of required.

Reply:
checked attribute cannot be used. user must first select the checkbox after reading terms and conditions.
required attribute must be used for data validation. the user must check the checkbox before submitting the form.

Reply:
You can do that, but my view is based on the sample project that we should follow and build similar to that sample. but you can choose that, which is not that much difference.

Reply:
we must follow design sample with critical thinking. we must think about proper data validation. we must use the required attribute because the user must select the checkbox before submitting because the device cannot be purchased without accepting terms.

we cannot use the checked attribute because the user must select the checkbox themselves. not doing so will be against rules and regulations regarding purchasing the product. we must use critical thinking when following the design. the user must select the checkbox after reading the terms. it is against purchasing rules and regulations for the user to not read the terms and conditions and then not select the checkbox and have it already selected.